lib/daru/index/multi_index.rb

```diff
@@ -14,6 +14,17 @@ def levels
       @levels.map(&:keys)
     end
 
+    def name=(names)
+      validate_name names, @labels
+      @name = names
+    end
```

Comment:
>You can (and, in fact, should) leave name= definition under the initialize, it will still work.

@zverok , are you saying to define `name=` below `initialize` or inside `initialize`?